src/components/Images/MyGallery.vue

```diff
@@ -32,17 +32,17 @@ const getThumbnails = async (sessionId) => {
   }
 
   await fetchApiCall({
-    url: configurationStore.thumbnailArchiveUrl + `thumbnails/?observation_id=${sessionId}&size=large`,
+    url: configurationStore.thumbnailArchiveUrl + `thumbnails/?observation_id=${requestId}&size=large`,
```

Comment:
I always welcome and encourage feedback. I also appreciate when it's done kindly and respectfully

Reply:
Can you try being more constructive and less judgmental on my cognition? Thanks!

Reply:
Sorry, I wasn't meaning to offend you, I could of phrased that better. I was just trying to point out that it should raise a red flag when you find yourself equating requests and observations, especially their ID's in a query, since those are two different constructs in our system with different IDs. It either means your variable name is wrong and you mean `observationId` but store it as `requestId`, or you are passing the wrong ID and won't get the expected results back.